crates/uv-virtualenv/src/activator/activate

```diff
@@ -24,16 +24,16 @@
 
 # Get script path (only used if environment is relocatable).
 if [ -n "${BASH_VERSION:+x}" ] ; then
-    SCRIPT_PATH="${BASH_SOURCE[0]}"
-    if [ "$SCRIPT_PATH" = "$0" ]; then
+    _UV_VENV_ACTIVATOR="${BASH_SOURCE[0]}"
```

Comment:
I'm not sure how the variable is actually used, but I think "for good measure", you should also unset it in `deactivate()` function, as in `unset _UV_VENV_ACTIVATOR`.

Otherwise, a de-activated venv could fool the Rust code below, it could make things look as if the venv is still active (based on the existence and value of the shell variable)